### PR TITLE
feat(Linear): Update docs to reflect new feature

### PR DESCRIPTION
**Pull request by ceorourke**
Linear has added the ability to automatically create issues via alert rules, so this PR updates the docs to reflect that.

### CODE REVIEW

File: src/docs/product/integrations/project-mgmt/linear/index.mdx [hunk 1]
```diff
@@ -25,3 +25,26 @@ Linear **won't** work with self-hosted Sentry.
    ![](linear-install-pg.png)
 
 2. Follow the full [Linear installation instructions](https://linear.app/docs/sentry).
+
+
+## Issue Management
+
+Issue tracking allows you to create Linear issues from within Sentry, and link Sentry issues to existing Linear issues. Issue management can be configured in two ways - automatically or manually.
```
Comment (imatwawana) [inline src/docs/product/integrations/project-mgmt/linear/index.mdx:32]: Changing en dash to em dash

```suggestion
Issue tracking allows you to create Linear issues from within Sentry, and link Sentry issues to existing Linear issues. Issue management can be configured in two ways — automatically or manually.
```

File: src/docs/product/integrations/project-mgmt/linear/index.mdx [hunk 2]
```diff
@@ -25,3 +25,26 @@ Linear **won't** work with self-hosted Sentry.
...
+
+#### Automatically
+
+To configure issue management automatically, create an [**Issue Alert**](/product/alerts/alert-types/#issue-alerts) or a [**Metric Alert**](/product/alerts/alert-types/#metric-alerts). When selecting the [**action**](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions), choose **Notify Integration** >> **Linear** for issue alerts, or select **Linear** from the action dropdown for metric alerts.
```
Comment (imatwawana) [inline src/docs/product/integrations/project-mgmt/linear/index.mdx:36]:
```suggestion
To configure issue management automatically, create an [issue alert](/product/alerts/alert-types/#issue-alerts) or a [metric alert](/product/alerts/alert-types/#metric-alerts). When selecting the [action](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions), choose "Notify Integration" and then "Linear" for issue alerts, or select "Linear" from the action dropdown for metric alerts.
```

File: src/docs/product/integrations/project-mgmt/linear/index.mdx [hunk 3]
```diff
@@ -25,3 +25,26 @@ Linear **won't** work with self-hosted Sentry.
...
+
+![](linear-rule.png)
+
+A Linear issue will be created automatically when the alert has been triggered.
```
Comment (imatwawana) [inline src/docs/product/integrations/project-mgmt/linear/index.mdx:40]:
```suggestion
A Linear issue will be created automatically when the alert is triggered.
```

File: src/docs/product/integrations/project-mgmt/linear/index.mdx [hunk 4]
```diff
@@ -25,3 +25,26 @@ Linear **won't** work with self-hosted Sentry.
    ![](linear-install-pg.png)
 
 2. Follow the full [Linear installation instructions](https://linear.app/docs/sentry).
+
+
+## Issue Management
+
+Issue tracking allows you to create Linear issues from within Sentry, and link Sentry issues to existing Linear issues. Issue management can be configured in two ways - automatically or manually.
+
+#### Automatically
```
Comment (imatwawana) [inline src/docs/product/integrations/project-mgmt/linear/index.mdx:34]: Always avoid skipping heading levels as this causes the right sidebar to not render properly.

```suggestion
### Automatically
```

File: src/docs/product/integrations/project-mgmt/linear/index.mdx [hunk 5]
```diff
@@ -25,3 +25,26 @@ Linear **won't** work with self-hosted Sentry.
...
+
+To configure issue management automatically, create an [**Issue Alert**](/product/alerts/alert-types/#issue-alerts) or a [**Metric Alert**](/product/alerts/alert-types/#metric-alerts). When selecting the [**action**](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions), choose **Notify Integration** >> **Linear** for issue alerts, or select **Linear** from the action dropdown for metric alerts.
+
+![](linear-rule.png)
+
+A Linear issue will be created automatically when the alert has been triggered.
+
+#### Manually
```
Comment (imatwawana) [inline src/docs/product/integrations/project-mgmt/linear/index.mdx:42]:
```suggestion
### Manually
```

File: src/docs/product/integrations/project-mgmt/linear/index.mdx [hunk 6]
```diff
@@ -25,3 +25,26 @@ Linear **won't** work with self-hosted Sentry.
...
+
+To configure issue management manually, once you've navigated to a specific Sentry issue, you'll find the **Linked Issues** section on the right hand panel.
```
Comment (imatwawana) [inline src/docs/product/integrations/project-mgmt/linear/index.mdx:44]:
```suggestion
To configure issue management manually, navigate to a specific Sentry issue and find the "Linked Issues" section on the right hand panel.
```

**Review by imatwawana — APPROVED** [review]
Mostly style fixes with a few small wording suggestions. Otherwise looks great!

**Review by NisanthanNanthakumar — APPROVED** [review]

**Pull request merged**